force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -94,16 +136,23 @@ public with sharing class DummyJSONCallout {
 	
 	public static void postCreateDummyJSONUser(String contactId) {
 		// Create HTTP request to send.
+		Http http = new Http();
+		HttpRequest request = new HttpRequest();
 
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+		request.setEndpoint('callout:DummyJsonUser.namedCredential');
 		// Set the HTTP method to POST.
-
+		request.setMethod('POST');
 		// Set the body using generateDummyJsonUserPayload method.
-
+		request.setBody(generateDummyJsonUserPayload(contactId));
 		// Send the HTTP request and get the response.
-
+		HttpResponse response = http.send(request);
 		// If the HTTP response code is successful, update the contact.
+		if(response.getStatusCode() >= 200 && response.getStatusCode() < 300) {
+			Contact cont = [SELECT Id, DummyJSON_Last_Updated__c FROM Contact WHERE Id = :contactId];
+			cont.DummyJSON_Last_Updated__c = Datetime.now();
+			update cont;
+		}
```

Comment:
This is a SOQL inside of a for loop. 
IF you are just updated the values without using the existing information then you can use a temp object with the id
```suggestion
		if(response.getStatusCode() >= 200 && response.getStatusCode() < 300) {
			Contact cont = new Contact(Id = contacted);
			cont.DummyJSON_Last_Updated__c = Datetime.now();
			update cont;
		}
```